code/parachain/frame/oracle/src/tests.rs

```diff
@@ -277,6 +280,10 @@ mod add_asset_and_info {
 				// and therefore this test should also be changed.
 				prop_assert_eq!(MaxAssetsCount::get(), 2_u32);
 
+
+				asset_info_1.reward_weight = asset_info_1.reward_weight.min(u128::MAX / 3);
```

Comment:
can we test with different weights?

Reply:
this is proptest, waits for 3 will be all different, i divided by 3 to make sure that they wont overflow. i can change it so that one is 1/12 of the max, another 1/6th of  the max , and third 3/4 of the max